workspace.dsl

```diff
@@ -0,0 +1,167 @@
+    					componentBoStateMachine = component "State machine" "Handles the appeal lifecycle and state transitions"
+    				}
+
+    				containerBoAzureSql = container "Database" "Data store for appeals" "Azure SQL"
+				}
+
+				group "Azure functions" {
+				    containerFunctionAppCasedata = container "Casedata import" "Consumes appellant and LPA submissions from Service Bus" "Function App, JavaScript"{
+						tags "Microsoft Azure - Function Apps", "FunctionApp"
+					}
+					containerFunctionAppDocs = container "Document processing" "Copies blobs from Front-Office, processes virus scanning" "Function App, JavaScript"{
+						tags "Microsoft Azure - Function Apps", "FunctionApp"
+					}
+				}
+			}
+
+			systemIntegration = softwareSystem "Integration Layer" "Event-driven message exchange between applications" {
+				tags "Microsoft Azure - Azure ServiceBus Namespace" "InternalCollaborationSystem"
+
+				containerBoServiceBus = container "Service Bus" "Messaging platform for data exchange" "Azure ServiceBus Namespace" {
+					tags "Microsoft Azure - Azure ServiceBus Namespace"
+
+					group "Commands" {
+						componentCaseCmdTopic = component "Service Bus Appeal submission topic" "Appeal submission commands"
+						componentLpaqCmdTopic = component "Service Bus LPA submission topic" "LPA submission commands"
+					}
+					group "Events" {
+						componentCaseMessageTopic = component "Service Bus Appeal topic" "Appeal messages"
+						componentDocumentMessageTopic = component "Service Bus Document topic" "Appeal document messages"
+						componentServiceUserMessageTopic = component "Service Bus Service User topic" "Service user messages"
+						componentEventMessageTopic = component "Service Bus Event topic" "Event message"
+					}
+				}
+
+				containerODW = container "Operational Data Warehouse (ODW)" "Holds all Planning Inspectorate data so that it can be used for internal purposes" {
+				  tags "Microsoft Azure - Azure Synapse Analytics" "InternalCollaborationSystem"
+				}
+
+				containerEventGrid = container "Azure Event Grid", "Reports virus scan results for uploaded blobs" {
+					tags "Microsoft Azure - Azure Event Grid" "InternalCollaborationSystem"
+				}
+			}
+
+			systemGovUk = softwareSystem "GOV Notify" {
+				tags = "ExternalSystem"
+				containerGovNotify = container "GOV Notify" "UK government messaging platform for sending emails, text and letters to users"
+			}
+
+			# Relationships
+
+			userCaseOfficer -> systemAppsBo "Manage case processing"
+			userInspector -> systemAppsBo "Arranges site visits"
+			userAppellant ->  systemAppsFo "Registers an appeal"
+			userLocalPlanningAuthority ->  systemAppsFo "Responds to an appeal"
+			containerGovNotify -> userLocalPlanningAuthority "Notifies the Local Planning Authority"
+			containerGovNotify -> userAppellant "Notifies the appellant"
+
+			systemAppsFo -> componentCaseCmdTopic "Records appellant submissions for the Back-Office to process"
```

Comment:
For this and other relationships relating to a Service Bus topic, you can add a tag of `ServiceBus` and it'll render these lines in a slightly different style on the diagram

i.e. 
`systemAppsFo -> componentCaseCmdTopic "Records appellant submissions for the Back-Office to process" "ServiceBus"`